lib/github_service/commands/cross_repo_test.rb

```diff
@@ -139,8 +140,8 @@ def parse_value(value)
         repo_groups, @test_repos = @test_repos.partition { |repo| repo.start_with?("/") }
         @test_repos |= expand_repo_groups(repo_groups)
 
-        # Add the identifier for the PR for this comment to repos
-        @repos << "#{issue.repo_name}##{issue.number}"
+        # Add the PR for this comment to the list of test repos
+        @test_repos << "#{issue.repo_name}##{issue.number}"
 
         # Add any test_repos that are branches or PRs to the repos list
         @repos |= @test_repos.select { |repo| branch_or_pr?(repo) }
```

Comment:
Note that since `@repos` is a super-set of `@test_repos` from here, we don't have to change the above to append to both and we only have to append to `@test_repos`